typings/css.d.ts

```diff
@@ -0,0 +1,9 @@
+declare function css(chunks: TemplateStringsArray, ...args: any[]): string
+declare namespace css {
+  function global(chunks: TemplateStringsArray, ...args: any[]): string
+  function resolve(
+    chunks: TemplateStringsArray,
+    ...args: any[]
+  ): { className: string; styles: string }
```

Comment:
I didn't have any issues while working with `@types/styled-jsx`, basically, I just copied over the types definitions.
It's weird because in the `css.js` file, it seems like `css.resolve`, `css.global` and `css` returns `void` (nothing).
I tested with the current types definitions like that :
```ts
import css from 'styled-jsx/css'

const button = css.resolve`
  button { color: hotpink; }
`

export default () => (
  <div>
    <button>styled-jsx</button>
    <style jsx>{button}</style>
  </div>
)
```
TypeScript doesn't show any errors or anything, so I guess we can let as it is now, we don't need to change it.

Reply:
right because a string is a valid JSX expression `{button}` .. but that's not what `css.resolve` returns

Reply:
Right, what does `css.resolve` returns ?
It returns JSX elements ?